app/src/main/java/com/github/polybooks/core/database/implementation/SaleDatabase.kt

```diff
@@ -220,13 +263,23 @@ class SaleDatabase : SaleDatabase {
     }
 
     override fun deleteSale(sale: Sale) {
+        if(sale.seller == LocalUser)
+            throw LocalUserException("Cannot add sale as LocalUser")
         SalesQuery().getReferenceID(sale).continueWith { task ->
-            val result = task.result.filter { document ->
+            Log.d("1===================================","${task.result.documents}" )
+            //println("1=========================${task.result.documents}")
+            val result = task.result.documents.filter { document ->
+                Log.d("0===================================","$document" )
                 val s = snapshotToSale(document)
-                s.condition == sale.condition && s.seller == sale.seller && s.date == sale.date
+                Log.d("2===================================","LEA IS HERE" )
+                Log.d("2===================================","$s" )
+                //println("2=========================$s")
+                s.condition == sale.condition /*&& s.seller == sale.seller */&& s.date == sale.date
```

Comment:
Pas sûre de savoir ce que c'est "LEA IS HERE", mais bon tu dois savoir ce que tu fais

Reply:
Hahaha les traces du debugging